lib/karafka/testing/rspec/helpers.rb

```diff
@@ -85,13 +85,15 @@ def _karafka_consumer_for(requested_topic, requested_consumer_group = nil)
         #     karafka.produce({ 'hello' => 'world' }.to_json, 'partition' => 6)
         #   end
         def _karafka_add_message_to_consumer_if_needed(message)
-          # We're interested in adding message to subject only when it is a consumer
+          # Consumer needs to be defined in order to pass messages to it
+          return unless respond_to?(:consumer)
```

Comment:
Maybe better to `defined?(:consumer)`, it will make local variables work.

Reply:
`defined?(:consumer)` always returns for me value `expression`, never false nor nil and this breaks the whole code as it always passes

Reply:
oh, it should be `defined?(consumer)`, not `:consumer`

Reply:
good point, works